content/concepts/kata/beta-process.md

```diff
@@ -5,98 +5,46 @@ sidebar: explanation-kata
-
-## Kata Approval
-
-After a kata has received enough approval votes from the community,
-fixed any issues, and reached a consensus on it's ranking,
-it's ready for the final step to come out of beta.
-
-At this point a moderator will review it to make sure all this criteria has been met,
-and release it as an official kata onto the site!
-
-As you get more katas approved, you'll become established as an experienced kata maker, and less votes will be needed for future katas to be approved.
-
-The following table lists the requirements for a beta kata to be able to be approved:
-
-<table>
-<thead>
-<tr>
-<th>Level</th>
-<th>Satisfaction %</th>
-<th colspan="4">Required Votes by # of Approved Katas</th>
-</tr>
-<tr>
-<th></th>
-<th></th>
-<th>&lt;10</th>
-<th>&lt;20</th>
-<th>&lt;30</th>
-<th>30+</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td>White</td>
-<td>80+</td>
-<td>12</td>
-<td>10</td>
-<td>8</td>
-<td>6</td>
-</tr>
-<tr>
-<td>Yellow</td>
-<td>80+</td>
-<td>10</td>
-<td>8</td>
-<td>7</td>
-<td>5</td>
-</tr>
-<tr>
-<td>Blue</td>
-<td>75+</td>
-<td>8</td>
-<td>6</td>
-<td>5</td>
-<td>4</td>
-</tr>
-<tr>
-<td>Purple</td>
-<td>70+</td>
-<td>5</td>
-<td>4</td>
-<td>3</td>
-<td>3</td>
-</tr>
-</tbody>
-</table>
-
-## Auto-Retire
-
-Low quality kata are automatically retired. Current rules are:
-
-1. If 4 or more votes and satisfaction percent is `0` (4+ consecutive "Not Satisfied")
-2. If `n` or more votes and satisfaction percent is less than or equal to the half of the required satisfaction (see above). `n` is the same as the number of votes needed to be approved (see above).
-
-This feature needs improvement and any feedback is appreciated. Please comment on [#1672](https://github.com/Codewars/codewars.com/issues/1672).
+Each kata is crafted for and by the community. Kata are created by other users rewarded with kata authoring privilege. When a kata is published by an author (or Sensei), it will go into a beta process where the community can help test and provide quality assurance feedback and cast a vote for its difficulty ranking. The feedback during this process is vital to making sure all kata are of the highest quality.
+
+## Reward for Reviewers
+
+You earn an _additional_ 2 honor for every Beta Kata you solve, in order to reward you for taking the risk of solving a Kata with potential issues. Then, leaving a satisfaction vote and rank assessment on said Beta Kata rewards you with 1 _extra_ honor _each_, in order to reward you for taking the time to leave valuable feedback on said Kata. Finally, if the Kata is approved, you will also receive the honor and rank progress for the official Kata. Therefore, it is in everyone's interest that you leave your satisfaction vote and rank assessment on every Beta Kata you have completed.
+
+## Steps of Beta Process
+
+Some of the steps are available only after earnig the required privilege, but every user can help reviewing a kata in some way.
```

Comment:
```suggestion
Some of the steps are available only after earning the required privilege, but every user can help reviewing a kata in some way.
```

@DonaldKellett Nice catch. You can make suggestions like this instead. See 6 in https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request

Reply:
Yeah sorry about the stray comment, I was still figuring out how the review process works.

Reply:
No need to be sorry. And I meant to use the suggestion that can be applied.

This thing:
![image](https://user-images.githubusercontent.com/639336/97250324-f23b5100-17c2-11eb-8825-a6a7fc3c6086.png)